puppet/modules/slave/manifests/postgresql.pp

```diff
@@ -1,22 +1,38 @@
 # @api private
 class slave::postgresql {
-  Postgresql_psql {
-    cwd => '/',
-  }
+  # only CentOS slaves are used to run unit tests
+  if $::osfamily == 'RedHat' {
+    if ::operatingsystemmajrelease == '7' {
+      include scl
 
-  include ::postgresql::client
-  include ::postgresql::server
+      scl::collection { 'rh-postgresql10':
+        enable => true,
+      }
 
-  # Simple known user/pass that will allow Jenkins to create its
-  # own databases when required
-  postgresql::server::role { 'foreman':
-    password_hash => postgresql_password('foreman', 'foreman'),
-    superuser     => true,
-    login         => true,
-  }
+      $non_scl_rpms = ['postgresql', 'postgresql-server', 'postgresql-devel', 'postgresql-libs']
+      $non_scl_rpms.each |String $non_scl_rpm| {
+        package { "non_scl_${non_scl_rpm}":
```

Comment:
You can also specify it at once. Probably also good to ensure it happens before postgresql is ran.
```ruby
package { $non_scl_rpms:
  ensure => absent,
  before => Class['postgresql::globals'],
}
```

Reply:
Nope...
```
Error: Evaluation Error: Error while evaluating a Resource Statement, Duplicate declaration: Package[postgresql-devel] is already declared at (file: /root/foreman-infra/puppet/modules/postgresql/manifests/lib/devel.pp, line: 21); cannot redeclare (file: /root/foreman-infra/puppet/modules/slave/manifests/postgresql.pp, line: 19) (file: /root/foreman-infra/puppet/modules/slave/manifests/postgresql.pp, line: 19, column: 1) on node localhost.localdomain
```